### PR TITLE
[BPK-4414] Move Flare View to Navigation Data

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Example/Backpack/NavigationData.swift [hunk 1]
```diff
@@ -169,7 +198,6 @@ class NavigationData: NSObject {
                     value: .story(loadStoryboard(name: "Gradients", identifier: "GradientViewController"))
                 )
             }
-            Item(name: "Gradients", value: .story(loadStoryboard(name: "Gradients", identifier: "GradientsViewController")))
```
Comment (georgegillams) [inline Example/Backpack/NavigationData.swift:172]: Why did this die?

**Review by georgegillams — CHANGES_REQUESTED** [review]
We can also remove `Example/Backpack/ViewControllers/FlareViewSelectorViewController.swift` now too, right?

File: Example/Backpack/NavigationData.swift (hunk 1 above)
Comment (k0nserv) [reply Example/Backpack/NavigationData.swift:172]: It was duplicated and old

**Comment by k0nserv** [comment]
> We can also remove Example/Backpack/ViewControllers/FlareViewSelectorViewController.swift now too, right?

Yes

**Review by georgegillams — APPROVED** [review]

**Pull request merged**